.github/workflows/validate-domain-library.yml

```diff
@@ -50,7 +50,23 @@ jobs:
     with:
       package-type: wheel
       os: linux
-      channel: ${{ inputs.channel || 'nightly' }}
+      channel: ${{ inputs.channel }}
+      with-cuda: disable
```

Comment:
is disable with-cuda intentional here?

Reply:
Yes. For now. Will add this as input parameter later